website/content/docs/installation.md

```diff
@@ -38,13 +38,9 @@ Homebrew also automatically installs completions for zsh and Bash.
 
 Install bloop in other platforms (Windows, Unix, \*bsd) via our python script:
 
-```sh
-$ curl -L https://github.com/scalacenter/bloop/releases/download/v1.0.0-M5/install.py | python2
-```
-
-<span class="label warning">Remember</span> to replace the version in the URL
-by the latest version, which is [![Latest
-version](https://index.scala-lang.org/scalacenter/bloop/sbt-bloop/latest.svg)](https://index.scala-lang.org/scalacenter/bloop/sbt-bloop).
+<pre><code class="language-sh">
+$ curl -L https://github.com/scalacenter/bloop/releases/download/<i class="latest-version">1.0.0-M7</i>/install.py | python2
```

Comment:
I meant here

Reply:
might wanna use a different default than `1.0.0-M7` in case people disable js for example

Reply:
We'll deal with that when we have tut. For now, we'll manually replace that for every release.